[STGMON-719] fix terraform and manual onboarding docs for the inventory prefix (#37601)
DataDog/documentation: content/en/infrastructure/storage_management/amazon_s3.md

}

module "datadog_storage_management" {
  source = "DataDog/storage-management-datadog/aws"

  name                              = "<MODULE_NAME>"
  datadog_aws_integration_role_name = "<DATADOG_AWS_INTEGRATION_ROLE_NAME>"
  source_bucket_names               = ["<SOURCE_BUCKET_1>", "<SOURCE_BUCKET_2>"]
  destination_bucket_name           = "<DESTINATION_BUCKET_NAME>"

  # Prefix within the destination bucket where inventory files are written.
  # Defaults to "datadog-inventories/". The module keeps the inventory
  # configuration, bucket policy, and IAM permissions consistent with this

>   # value, so leave it at the default unless you have a specific reason to

Reviewer: ```suggestion
  # value. Datadog recommends to use the default value.
```

>   # change it.

Reviewer: ```suggestion
```

1. [Create an S3 bucket][201] to store your inventory files. This bucket acts as the central location for inventory reports.
   **Note**: Use only one destination bucket for all inventory files generated in an AWS account.

> 2. Decide on the destination prefix within the bucket where inventory files are written. The standard prefix is `datadog-inventories/`, which is also the default if you leave the value empty. To read inventory reports from the entire bucket, set the value to `/`. You can use a custom prefix, but you must use the same value in every later step: the S3 Inventory configuration, the destination bucket policy, the integration role permissions, and the API registration call. Mismatched prefixes prevent Datadog from reading your inventory files.

Reviewer: ```suggestion
2. Decide on the destination prefix within the bucket where inventory files are written. The standard prefix is `datadog-inventories/`, which is also the default if you leave the value empty. To read inventory reports from the entire bucket, set the value to `/`. You can use a custom prefix, but you must use the same value in subsequent steps: the S3 Inventory configuration, the destination bucket policy, the integration role permissions, and the API registration call. Mismatched prefixes prevent Datadog from reading your inventory files.
```

     {{< img src="integrations/guide/storage_monitoring/all-versions.png" alt="Select destination buckets for enabling Storage Monitoring" responsive="true">}}
   - **Destination**: Select the common destination bucket for inventory files in your AWS account. For example, if the bucket is named `destination-bucket`, enter `s3://your-destination-bucket`

>       **Note**: Add the destination prefix you chose in step 1. Use the same value here that you register with Datadog in the post-setup step.

Reviewer: ```suggestion
      **Note**: Add the destination prefix you chose in step 1. Use the same value that you register with Datadog in the post-setup step.
```
